src/xdp-utils.c

```diff
@@ -163,16 +164,24 @@ _xdp_parse_app_id_from_unit_name (const char *unit)
   /*
    * From https://systemd.io/DESKTOP_ENVIRONMENTS/ the format is one of:
    * app[-<launcher>]-<ApplicationID>-<RANDOM>.scope
+   * app[-<launcher>]-<ApplicationID>-<RANDOM>.slice
```

Comment:
No idea tbh. The document says

> TODO: Define the name of slices that should be used. This could be app-\<launcher\>-
<ApplicationID\>-\<RANDOM\>.slice.

So I very much hope nothing uses it but I also don't think matching on it makes anything worse.

Reply:
Yeah, certainly not a blocker. If an app process does somehow exist in `app[-<launcher>]-<ApplicationID>-<RANDOM>.slice` then its app ID is clearly meant to be `<ApplicationID>`.